2주차/할리갈리.py

```diff
@@ -0,0 +1,19 @@
+number = int(input())
+
+S = {'STRAWBERRY': 0, 'BANANA': 0, 'LIME': 0, 'PLUM': 0}
+
+for _ in range(number):
+    card = input().upper().split()
+    if card[0] in S:
+        S[card[0]] += int(card[1])
+
+found = False
+for key in S:
+    if S[key] == 5:
+        found = True
+        break
+
+if found:
+    print('YES')
+else:
+    print('NO')
```

Comment:
```suggestion
for key in S:
    if S[key] == 5:
        print('YES')
        break
else:
    print('NO')
```
flag 변수 사용하기보단 파이썬스럽게  for-else문 사용해봅시다